app/api/posts/route.ts

```
@@ -5,7 +5,61 @@
 import { NextResponse } from 'next/server';
 import { getAllPosts } from '@/lib/blog';
 
+/**
+ * Formats error details for API response
+ * In production, this will omit sensitive information
+ */
+function formatErrorResponse(error: unknown) {
+  const isDev = process.env.NODE_ENV === 'development';
+
+  // Basic error info for any environment
+  const errorResponse: Record<string, any> = {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)